jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/navigation/BreadcrumbsTests.java

```diff
@@ -3,116 +3,96 @@
 
-        breadcrumbsFrame.breadcrumb.getSeparatorByIndex(1).is().text("›");
-        breadcrumbsFrame.breadcrumb.getSeparatorByIndex(2).is().text("›");
-        breadcrumbsFrame.breadcrumb.getSeparatorByIndex(3).is().text("-");
-        breadcrumbsFrame.breadcrumb.getSeparatorByIndex(4).is().text("-");
+    @Test
+    public void withIconsBreadcrumbsTest() {
+        simpleBreadcrumbsPage.open();
+
+        icons.get(1).is().displayed();
+        icons.get(2).is().displayed();
+        icons.get(3).is().displayed();
     }
 
     @Test
-    public void withIconsBreadcrumbsTest(){
-        withIconsBreadcrumbsPage.open();
+    public void collapsedBreadcrumbsTest(){
+        simpleBreadcrumbsPage.open();
+
+        breadcrumbs.get(16).is().displayed();
+        breadcrumbs.get(16).is().text("Home");
+        breadcrumbs.get(17).is().text("Belts");
 
-        commonCheck();
+        collapsedButton.is().displayed();
+        collapsedButton.click();
 
-        breadcrumbsFrame.breadcrumb.getIconByIndex(1).is().displayed();
-        breadcrumbsFrame.breadcrumb.getIconByIndex(2).is().displayed();
-        breadcrumbsFrame.breadcrumb.getIconByIndex(3).is().displayed();
+        breadcrumbs.get(17).is().text("Catalog");
+
+        collapsedButton.is().notVisible();
     }
 
     @Test
-    public void collapsedBreadcrumbsTest(){
-        collapsedBreadcrumbsPage.open();
-
-        breadcrumbsFrame.breadcrumb.getBreadcrumbByIndex(1).is().text("Home");
-        breadcrumbsFrame.breadcrumb.getBreadcrumbByIndex(1).click();
-        breadcrumbsFrame.breadcrumb.getBreadcrumbByIndex(2).is().text("Belts");
-        breadcrumbsFrame.breadcrumb.getBreadcrumbByIndex(2).click();
-
-        breadcrumbsFrame.collapsedButton.is().displayed();
-        breadcrumbsFrame.collapsedButton.click();
-
-        breadcrumbsFrame.breadcrumb.getBreadcrumbByIndex(1).is().text("Home");
-        breadcrumbsFrame.breadcrumb.getBreadcrumbByIndex(1).click();
-        breadcrumbsFrame.breadcrumb.getBreadcrumbByIndex(2).is().text("Catalog");
-        breadcrumbsFrame.breadcrumb.getBreadcrumbByIndex(2).click();
-        breadcrumbsFrame.breadcrumb.getBreadcrumbByIndex(3).is().text("Accessories");
-        breadcrumbsFrame.breadcrumb.getBreadcrumbByIndex(3).click();
-        breadcrumbsFrame.breadcrumb.getBreadcrumbByIndex(4).is().text("New Collection");
-        breadcrumbsFrame.breadcrumb.getBreadcrumbByIndex(4).click();
-        breadcrumbsFrame.breadcrumb.getBreadcrumbByIndex(5).is().text("Belts");
-        breadcrumbsFrame.breadcrumb.getBreadcrumbByIndex(5).click();
-
-        breadcrumbsFrame.collapsedButton.is().notVisible();
+    public void customizedBreadcrumbsTest(){
+        simpleBreadcrumbsPage.open();
+
+        breadcrumbs.get(18).is().text("Home");
+        breadcrumbs.get(18).click();
+        breadcrumbs.get(19).is().text("Catalog");
+        breadcrumbs.get(19).click();
+        breadcrumbs.get(20).is().text("Accessories");
+        breadcrumbs.get(20).click();
```

Comment:
For page simpleBreadcrumbsPage should be checked e.g. click on "Core" beradcrumb and then that it's not visible Material UI text at the bottom. Enough 2-3 checks for different text